docs/custom-metric-filter.md

```diff
@@ -0,0 +1,195 @@
+# ACM Custom Metric Filtering
+
+TL;DR: This file provides guides over adjusting the custom metrics which Config Sync exports to Cloud Monitoring, and workarounds for enabling/disabling
```

Comment:
you can drop the "TL;DR:"

Reply:
How about: 

"This guide explains how to adjust the custom metrics that Config Sync exports to Prometheus, Cloud Monitoring (formerly known as Stackdriver), and Monarch (Google's internal metrics aggregator)."

Monarch has been talked about publicly before, so it's safe to mention. We should also explain the difference between the customer facing metrics and the internal metrics later in the doc.

Reply:
Done.